frontend/js/components/battle-details/BattleInfoDetails.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable babel/camelcase */
 import get from 'lodash/get';
```

Comment:
It's suggested to use destructuring import here because it's quite often to import more than one function from lodash, and doing so you're able to make all imports at the same line instead of having one line for each imported function.

```suggestion
import { get } from 'lodash';
```